mothership-ec/cog-mothership-commerce
# Feature/multiple currencies

#### What does this do?

Adds the potential for multiple currencies to mothership. It allows multiple prices to be set agains t products and units, and allows users to switch which currency they are viewing the site with. 
#### How should this be manually tested?

Setup the currency switcher - add:

```
{{ render(controller('Message:Mothership:Commerce::Controller:Module:CurrencySelect#form')) }}
```

to a template to view it.

The currencies config file will need to be added to the test site. The setup is in fixtures/.

Try adding multiple currencies to both product and unit. Make sure they save and are viewable on the front end using the currency switcher.

The price and unit forms have been remade, make sure all functionality of the forms work (not just the price changes).

NOTE: There is a bug on tonic where product not unit prices are shown on product pages. The basket shows the correct price for units though.

Test on union without config files. Make sure the lack of config files doesn't break the site.
#### Related PRs / Issues / Resources?
#### Anything else to add? (Screenshots, background context, etc)
